chai-react.js

```diff
@@ -3,24 +3,23 @@
   if (typeof require === "function" && typeof exports === "object" && typeof module === "object") {
     // NodeJS
     module.exports = function (chai, utils) {
-      return chaiReact(chai, utils, require('react'));
+      return chaiReact(chai, utils, require('react'), require('react-addons-test-utils'));
     };
   } else if (typeof define === "function" && define.amd) {
     // AMD
-    define(['react'], function (React) {
+    define(['react', 'react-dom', 'react-addons-test-utils'], function (React, TestUtils) {
```

Comment:
Ah good catch. I initially added it but then realized I didn't need `react-dom`.